modules/virt-supported-ssp-tasks.adoc

```diff
@@ -37,3 +40,8 @@ The following table shows the tasks that are included as part of the Tekton Task
 | `wait-for-vmi-status`
 | Wait for a specific status of a virtual machine instance and fail or succeed based on the status.
 |===
+
+[NOTE]
+====
+Virtual machine creation in pipelines now utilize `ClusterInstanceType` and `ClusterPreference` instead of template-based tasks which have been deprecated. Commands `create-vm-from-template`, `copy-template`, and `modify-vm-template` remain available but are not used in default pipeline tasks.
```

Comment:
```suggestion
Virtual machine creation in pipelines now utilizes `ClusterInstanceType` and `ClusterPreference` instead of template-based tasks, which have been deprecated. The `create-vm-from-template`, `copy-template`, and `modify-vm-template` commands remain available but are not used in default pipeline tasks.
```